.github/workflows/build.yml

```
@@ -100,6 +105,18 @@ jobs:
         run: |
           pass init 7D851EB72D73BDA0
         shell: bash
+      -
+        name: Init gopass
+        run: |
+          gopass config mounts.path /root/.gopass-password-store 1>/dev/null
```

Don't think root is right, I guess `$HOME` or `~` is fine:

```suggestion
          gopass config mounts.path $HOME/.gopass-password-store 1>/dev/null
```

---

hmm, that could be it. i forget why i was explicitly setting it to `/root/...` in the first place.

---

that won't solve the windows test, but just sent another commit up to see if this fixes the tests for ubuntu and macos.

---

i don't think it will solve the windows test, but just sent another commit up to see if this fixes the tests for ubuntu and macos.